RELEASING.md

```diff
@@ -0,0 +1,315 @@
+# Guide to release Apache StormCrawler (Incubating)
+
+## Release Preparation
+
+- Elect a release manager. This can also be done via lazy consensus on the mailing list.
+- Create an issue for a new release in https://github.com/apache/incubator-stormcrawler/issues  If you do not have permission to do so just ask to be given permissions on the mailing list.
```

Comment:
The release manager should be at least a committer. If they do not have permissions to create issues on GitHub then they have not mapped their apache id to their GitHub id. This mapping also requires 2FA be enabled in the user's GitHub account.

Reply:
I think, that this is an issue from copy pasting. I rewrote this section. Everybody with a GitHub account can create an issue.